pages/data.py

```diff
@@ -116,9 +135,8 @@ def render_content(tab, store_uuids, store_trips, store_demographics, store_traj
         return None
 
     df = df.drop(columns=[col for col in df.columns if col not in columns])
-    df = clean_location_data(df)
 
-    return populate_datatable(df)
+    return populate_datatable(df, 'datatable')
```

Comment:
The string is just the table_id to pass some hidden columns inside datatable, I have used trips-datatable for trips table, other table share same filter so I used datatable for other tables.

Reply:
I don't understand this comment. where is this used to pass "hidden columns inside datatable"? I have searched through the code, and I don't see any other instances of this being used. If this is not used, does it need to be passed in?

Also, it is not clear to me if you used the same table_id for multiple tables. But if you did, why? Why not use separate names for each datatable to clarify the code instead of leading to potential issues with reusing the table ID.

Reply:
I will go ahead and update the name, so there is no more confusion.

Reply:
@achasmita updating the name only addresses the second part of this comment. I still don't see an answer to the first part.

Reply:
I have passed hidden columns on ```trips-datatable``` only. To pass the hidden column to trips table we need to provide id of the table and that is why I updated ```trips-datatable``` for trips table and ```datatable``` for other remaining table. And now have changed id for all table to make it more clear.